go/vt/servenv/pprof_windows.go

```diff
@@ -0,0 +1,6 @@
+//go:build windows
+
```

Comment:
Missing license. I also wonder why we log warnings in some places but not others. Probably worth logging a warning in all cases.